### PR TITLE
Update Smart Proxy installations instructions on Windows for DHCP proxy

### PR DESCRIPTION
**Pull request by claer**
The installation has been tested ok, the service registration fails and had to be corrected later on.

### CODE REVIEW

File: _includes/manuals/1.8/4.3.5.3_ms_dhcp.md [hunk 1]
```diff
@@ -6,130 +6,27 @@ It is required that this procedure is executed as an administrator.
 It is not required that the smart-proxy be on the same host as the MS dhcp server.  The smart-proxy just needs to be on a windows host that has netsh commands available.
 
 1. Go to the smart-proxy repository at https://github.com/theforeman/smart-proxy
```
Comment (stbenjam) [inline _includes/manuals/1.8/4.3.5.3_ms_dhcp.md]: Maybe link to https://github.com/theforeman/smart-proxy/releases directly

File: _includes/manuals/1.8/4.3.5.3_ms_dhcp.md [hunk 2]
```diff
@@ -155,22 +44,46 @@ _Sample config/settings.yml file_
...
+ # The dhcp_server is only used by the native_ms implementation
+ :dhcp_server: 10.10.10.1
+</pre>
+
+12. Create the SSL key
+
+12.1. Login to your puppetmaster
 
-10.2) puppetca --generate _Smart-proxy FQDN_. (Do not use an alias here.)
+12.2. On the command line, type the following command. Take care to ot use an alias nor upper case characters.
```
Comment (stbenjam) [inline _includes/manuals/1.8/4.3.5.3_ms_dhcp.md]: not to*

File: _includes/manuals/1.8/4.3.5.3_ms_dhcp.md [hunk 3]
```diff
@@ -5,131 +5,28 @@ It is required that this procedure is executed as an administrator.
...
-9) Edit config/settings.yml so that it looks a bit like this
+1. Go to the smart-proxy repository at https://github.com/theforeman/smart-proxy/releases
+2. Just under the tag matching your Foreman version, click the ZIP link in order to download the Smart Proxy source code
+3. Extract the archive to a directory that does not have any spaces in its name
+4. Go to the rubyinstaller web page at http://rubyinstaller.org/downloads/
+5. Download and install the "ruby 1.9.3-p551": (Allow the ruby associations to be installed.)
+6. Download and install the Development Kit For Ruby 1.9.3: (DevKit-tdm-32-4.5.2-20111229-1559-sfx.exe)
+7. Open a CMD window and, install the bundler_ext gem using *gem install --platform x86-mingw32 bundler_ext*
+8. CD to the root of the smart-proxy directory (the one you created in step 3)
+9. Copy the file Gemfile to GemFile.in : *copy Gemfile GemFile.in*
```
Comment (domcleal) [inline _includes/manuals/1.8/4.3.5.3_ms_dhcp.md:16]: I think you can leave this step out, and shouldn't need to install bundler_ext - just bundler.

On step 10 you use bundler, which should be the right way of doing it, we tend to only use bundler_ext when packaging gems as debs/rpms.

Edit: though you may want to double check it still works after deleting Gemfile.in

File: _includes/manuals/1.8/4.3.5.3_ms_dhcp.md [hunk 4]
```diff
@@ -155,22 +44,46 @@ _Sample config/settings.yml file_
...
+ # Enable DHCP management
+ :dhcp: true
+ # The vendor can be either isc or native_ms
+ :dhcp_vendor: native_ms
+ # The dhcp_server is only used by the native_ms implementation
+ :dhcp_server: 10.10.10.1
 </pre>
 
-10) Create the SSL key
+12. Create the SSL key
```
Comment (domcleal) [inline _includes/manuals/1.8/4.3.5.3_ms_dhcp.md]: You might have to use `12)` here as Markdown tries to restart the numbering from 1

File: _includes/manuals/1.8/4.3.5.3_ms_dhcp.md [hunk 5]
```diff
@@ -155,22 +44,46 @@ _Sample config/settings.yml file_
...
-10.2) puppetca --generate _Smart-proxy FQDN_. (Do not use an alias here.)
+12.2. On the command line, type the following command. Take care not to use an alias nor upper case characters.
+<pre>
+ # puppet ca generate _Smart-proxy FQDN_
+</pre>
+
+12.3. Copy the private key, the public certificate and the ca.pem from /var/lib/puppet/ssl over to the locations that you specified in the setting file.
 
-10.3) Copy the private key, the public certificate and the ca.pem from /var/lib/puppet/ssl over to the locations that you specified in the setting file.
+13. Test the installation by running ruby bin\smart-proxy.rb within an administrator cmd.exe prompt.
```
Comment (domcleal) [inline _includes/manuals/1.8/4.3.5.3_ms_dhcp.md]: ditto here and below

**Comment by mmoll** [comment]
could you squash the commits, please? :)

**Comment by mmoll** [comment]
merci @claer! :ice_cream:

**Pull request merged**